introduction.md

```diff
@@ -1,3 +1,8 @@
 Hello world.
 
-Favorite code school path: Ruby.
\ No newline at end of file
+Favorite code school path: Ruby.
+
+* Git
+* Java
+* Ruby
+* Unix
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?